studio-api/components/WebServer/routecontrollers/publication/publication.js

```diff
@@ -0,0 +1,467 @@
+const debug = require("debug")(
```

Comment:
Refactor `studio-api/components/WebServer/routecontrollers/publication/publication.js` to behave like a proxy, limiting responsibilities to only what is required.  

Reference similar logic in:  
- `components/WebServer/routes/router.js` (around line 177: `createProxyRoutes`)  
- `components/WebServer/routes/proxy/sessions/session.js`